pkg/platform/model/buildplanner.go

```diff
@@ -100,29 +100,26 @@ func NewBuildPlannerModel(auth *authentication.Auth) *BuildPlanner {
 	}
 }
 
-func (bp *BuildPlanner) FetchBuildResult(commitID strfmt.UUID, owner, project string) (*BuildResult, error) {
+func (bp *BuildPlanner) FetchBuildResult(commitID strfmt.UUID) (*BuildResult, error) {
 	resp := &bpModel.BuildPlan{}
-	err := bp.client.Run(request.BuildPlan(owner, project, commitID.String()), resp)
+	err := bp.client.Run(request.BuildPlan(commitID.String()), resp)
 	if err != nil {
 		return nil, errs.Wrap(err, "failed to fetch build plan")
 	}
 
 	// Check for errors in the response
-	if resp.Project.Type == bpModel.NotFound {
-		return nil, locale.NewError("err_buildplanner_project_not_found", "Build plan does not contain project")
-	}
-	if resp.Project.Commit.Type == bpModel.NotFound {
+	if resp.Commit.Type == bpModel.NotFound {
 		return nil, locale.NewError("err_buildplanner_commit_not_found", "Build plan does not contain commit")
 	}
-	if resp.Project.Commit.Build.Type == bpModel.BuildResultPlanningError {
+	if resp.Commit.Build.Type == bpModel.BuildResultPlanningError {
 		var errs []string
 		var isTransient bool
-		for _, se := range resp.Project.Commit.Build.SubErrors {
+		for _, se := range resp.Commit.Build.SubErrors {
 			errs = append(errs, se.Message)
 			isTransient = se.IsTransient
 		}
 		return nil, &BuildPlannerError{
-			wrapped:          locale.NewError("err_buildplanner", resp.Project.Commit.Build.Error),
+			wrapped:          locale.NewError("err_buildplanner", resp.Commit.Build.Error),
```

Comment:
This feels like it should be an input error.

I didn't really think about this when originally reviewing, but we want to make sure we don't end up logging build errors and solve errors to our rollbar, as they are not our responsibility.